src/styles/_academy.scss

```diff
@@ -189,6 +189,11 @@
     flex-direction: column;
   }
 
+  .listing-xp {
+    display: flex;
+    gap: 0.5rem;
+  }
```

Comment:
Can we start to move newly created styles to a separate CSS module for better abstraction (instead of using global CSS)?